recipes/recipes_emscripten/llvm/build.sh

```diff
@@ -0,0 +1,29 @@
+mkdir build
+cd build
+
+export CMAKE_PREFIX_PATH=$PREFIX
+export CMAKE_SYSTEM_PREFIX_PATH=$PREFIX
+
+# clear LDFLAGS flags because they contain sWASM_BIGINT
+export LDFLAGS=""
+
+
+
+# Configure step
+emcmake cmake ${CMAKE_ARGS} -S ../llvm -B .                 \
+    -GNinja                                         \
+    -DCMAKE_BUILD_TYPE=MinSizeRel                   \
+    -DCMAKE_PREFIX_PATH=$PREFIX                     \
+    -DCMAKE_INSTALL_PREFIX=$PREFIX                  \
+    -DLLVM_HOST_TRIPLE=wasm32-unknown-emscripten    \
+    -DLLVM_TARGETS_TO_BUILD="WebAssembly"           \
+    -DLLVM_INCLUDE_BENCHMARKS=OFF                   \
+    -DLLVM_INCLUDE_EXAMPLES=OFF                     \
+    -DLLVM_INCLUDE_TESTS=OFF                        \
+    -DCMAKE_CXX_FLAGS="-Dwait4=__syscall_wait4"
```

Comment:
The [conda-forge llvm build](https://github.com/conda-forge/llvmdev-feedstock/blob/main/recipe/build.sh#L30) also disables libedit `DLLVM_ENABLE_LIBEDIT=OFF`. Maybe this could help reduce the size a bit more?